appeals/web/src/server/appeals/appeal-details/final-comments/redact/redact.controller.js

```diff
@@ -0,0 +1,129 @@
+import logger from '#lib/logger.js';
+import { redactFinalCommentPage, confirmRedactFinalCommentPage } from './redact.mapper.js';
+import { patchFinalCommentRedaction } from './redact.service.js';
+import { patchFinalCommentsStatus } from '../view-and-review/view-and-review.service.js';
+import { COMMENT_STATUS } from '@pins/appeals/constants/common.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderRedactFinalComment = (request, response) => {
+	const {
+		errors,
+		currentRepresentation,
+		currentAppeal,
+		session,
+		params: { finalCommentsType }
+	} = request;
+
+	if (!currentRepresentation || !currentAppeal) {
+		logger.warn('No representation or appeal found in session');
+		return response.status(404).render('app/404.njk');
+	}
```

Comment:
Is this just for type safety? I feel like the middleware for loading these from the API should handle 404ing if either of these conditions happen

Reply:
No this was another habit of mine from before we were using middleware as much, I've removed these now as they don't serve any purpose